<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetTilesetSettings</class>
 <widget class="QWidget" name="WidgetTilesetSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Praticable</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="WidgetTilesetPraticable" name="widgetTilesetPraticable">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>374</width>
              <height>148</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBoxZoom">
           <property name="title">
            <string>Zoom</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QSlider" name="horizontalSlider">
              <property name="minimum">
               <number>-5</number>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="sliderPosition">
               <number>0</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="invertedControls">
               <bool>false</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxRepeat">
           <property name="text">
            <string>Repeat</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabDirections">
        <attribute name="title">
         <string>Directions</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="WidgetTilesetDirection" name="widgetTilesetDirection">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>374</width>
              <height>247</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetTilesetPraticable</class>
   <extends>QWidget</extends>
   <header>widgettilesetpraticable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetTilesetDirection</class>
   <extends>QWidget</extends>
   <header>widgettilesetdirection.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
